src/main/java/com/hms_networks/americas/sc/extensions/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -70,12 +69,29 @@ public static synchronized long getQueueFifoTimeSpanMins() {
   /**
    * Set the FIFO queue time span in minutes.
    *
+   * @deprecated
    * @param timeSpanMins new FIFO queue time span in minutes
    */
   public static synchronized void setQueueFifoTimeSpanMins(long timeSpanMins) {
     queueFifoTimeSpanMins = timeSpanMins;
   }
 
+  /**
```

Comment:
I thought I had commented on this before, but I don't see it anymore....


I don't think we should be introducing setQueueFifoTimeSpanMinsCheck(). It's redundant and the existing setQueueFifoTimeSpanMins() method should be updated. By changing the method name, it is just adding work in all connectors when they update to this version. 

I actually see this more as a bug fix, because adding the constraint should not affect any existing implementations, and if it does, they were using it improperly to begin with, so the exception may be a good thing so that it can get noticed/caught.

Reply:
ok. We can make the change. You are correct, all applications that this effects aren't using it properly.